web/documentserver-example/php/templates/index.tpl

```diff
@@ -140,6 +140,15 @@
             <span id="step1" class="step">1. Loading the file.</span>
             <span class="step-descr">The loading speed depends on file size
                         and additional elements it contains.</span>
+            <div id="xmlError" class="invisible">
+                <br />
+                <span class="step">Please select the current document type</span>
+                <div class="buttonsMobile indent">
+                    <div id="forceConvert" class="button forceConvert document" data="docx">Document</div>
```

Comment:
According to the HTML standard, an ID must be unique throughout the document.
https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id